Tasks/IISWebAppManagementOnMachineGroup/IISWebAppManagementOnMachineGroup.ps1

```diff
@@ -78,6 +79,12 @@ $appPoolPassword = Get-VstsInput -Name "AppPoolPassword"
 # Additional appcmd inputs 
 $appCmdCommands = Get-VstsInput -Name "AppCmdCommands"
 
+if($enableIIS -eq "true")
+{
+    import-module servermanager
```

Comment:
Please note that this task can be run on older version of windows OS as well containing older version of PS. they might not have ServerManager module installed. please test against different OSs

Reply:
Also if you need to install, then you should take care whether it should be system wide or just local

Reply:
Testing for multiple OS and Powershell version is in progress
It is just an import (not install) which doesn't have anything to do with specific user or system wide thing.

Reply:
test on Server2012 as well